Makefile

```diff
@@ -268,20 +268,41 @@ go_ldflags := '${go_ldflags}'
 
 .PHONY: build
 
-build: tidy bin/spire-server bin/spire-agent bin/k8s-workload-registrar bin/oidc-discovery-provider
+build: tidy bin/spire-server bin/spire-agent bin/k8s-workload-registrar bin/oidc-discovery-provider bin/spire-server-static bin/spire-agent-static bin/k8s-workload-registrar-static bin/oidc-discovery-provider-static
```

Comment:
Would you mind excluding the static binaries from the `build` target? I don't see a use for them today outside of the scratch image.

I think a separate `build-static` target would be appropriate for these binaries.